### PR TITLE
minor changes for consistency and readability

### PR DESCRIPTION
**Pull request by nlharris**
First I fixed a typo (it's -> its) but then I started making the list of operations more consistent. 

I removed "ontology term matching" after first changing it to "find matching terms" and concluding that was basically the same as "search an ontology for a term", but maybe there's something I'm missing.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,15 +9,14 @@ Python lib for common ontology operations over a variety of backends.
...
 
-This library provides a collection of different [interfaces](https://incatools.github.io/ontology-access-kit/interfaces/index.html) for different kinds of ontology operations, including:
+OAK provides a collection of [interfaces](https://incatools.github.io/ontology-access-kit/interfaces/index.html) for various ontology operations, including:
 
- - [lookup of basic features](https://incatools.github.io/ontology-access-kit/interfaces/basic.html) of an ontology element, such as it's label, definition, relationships, or aliases
+ - [look up basic features](https://incatools.github.io/ontology-access-kit/interfaces/basic.html) of an ontology element, such as its label, definition, relationships, or aliases
  - search an ontology for a term
- - validating an ontology
- - updating, deleting, or modifying terms
- - ontology term matching
```
Comment (matentzn) [inline README.md:18]: You dropped this? Why?

Comment (nlharris) [reply README.md:18]: I didn't drop those lines, I changed them for consistency! The changes show up as dropped lines and added lines.

Comment (matentzn) [reply README.md:18]: I cant find this specific line.. This seems to be removed without replacement.

Comment (nlharris) [reply README.md:18]: I removed "ontology term matching" after first changing it to "find matching terms" and concluding that was basically the same as "search an ontology for a term", but maybe there's something I'm missing.

Comment (matentzn) [reply README.md:18]: No no, very different: OAK has an ontology term matching module which results in a mapping table in SSSOM format! Different than search!

```
identify lexical matches and export as SSSOM mapping tables
```

**Review by matentzn — APPROVED** [review]

**Comment by nlharris** [comment]
Thanks for your helpful comments, Nico.
I am not empowered to merge or request a reviewer, but perhaps @pkalita-lbl can review and merge.

**Pull request merged**